project/util/address_form_fields.py

```diff
@@ -100,6 +101,16 @@ class AddressAndBoroughFormMixin(forms.Form):
         choices=BOROUGH_CHOICES.choices, required=False, help_text="A New York City borough."
     )
 
+    # Our onboarding scaffolding structure uses slightly different field names for the
+    # same kind of values; this dictionary maps our field names to the ones the
```

Comment:
instead of "maps *our* field names", which is a bit unclear, consider replacing with "maps form's field names" or something else more descriptive

Reply:
ah good idea, fixed!